src/components/package/edit-custom/custom-package-edit.js

```diff
@@ -43,14 +43,24 @@ class CustomPackageEdit extends Component {
     showSelectionModal: false,
     allowFormToSubmit: false,
     packageSelected: this.props.initialValues.isSelected,
+    packageVisible: this.props.initialValues.isVisible,
     formValues: {}
   }
 
   componentWillReceiveProps(nextProps) {
     let wasPending = this.props.model.update.isPending && !nextProps.model.update.isPending;
-    let needsUpdate = !isEqual(this.props.initialValues, nextProps.initialValues);
+    let needsUpdate = !isEqual(this.props.model, nextProps.model);
```

Comment:
curious about why this should be changed from initialValues to model because my understanding is that initialvalues are being populated from model, but per our discussion, we can clarify with @cherewaty later. Not stopping this PR